### ISSUE TITLE
bug: layout is weird after changing dpj

### ISSUE THREAD

**Issue by Amby713** [crdroidandroid/issue_tracker#36]
### Issue type

User interface

### Device

Mi 10t

### crDroid version

crDroid 8

### Exact version

8.10

### Bug description

I changed my dpi to 240 (termux > wm density 240) and noticed that when in landscape, the keyboard is small and stuck on the bottom right.

Lock screen pattern does the same thing in portrait mode

### Steps to reproduce

Change density
Lock screen and use pattern
Or
Use anything in landscape mode and click on text box to set keyboard get set to bottom right

### Relevant log (logcat/build log)

```shell
N/a
```

### Screenshots or videos

![Screenshot_20221112-171419_Bromite](https://user-images.githubusercontent.com/105613369/201498355-ff7cdf08-0dc3-41ab-8045-c161dcc04c91.png)
![Screenshot_20221112-171602_Bromite](https://user-images.githubusercontent.com/105613369/201498359-e7654175-92af-4ca0-878d-df847bd40cba.png)

### Acknowledgements

- [X] I've checked device is officially supported.
- [X] I'm running latest version available on crdroid.net for this device
- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by gwolf2u** [comment]
using non standard DPI is not quite the best approach here

**Comment by Amby713** [comment]
Yeah, but that's how I've been using it for years with no issue. I upgraded up 8.10, clean format install, and it was working perfectly. But i had to do another clean flash to troubleshoot the usb audio not working. I the second flash, also clean format, it went back to having that bar. So I know it can go away, just don't know how to

**Comment by gwolf2u** [comment]
Any mods installed?

**Comment by Amby713** [comment]
I have magisk, lsposed, and app settings reborn. Was using settings reborn, but now that glitches the apps, that is set specific dpi for,  and breaks other settings. It's currently not changing any of the apps right now.

**Comment by Amby713** [comment]
Tried that and it was the same. But I suppose I can try that again tonight. 

Here's my bottom nav bar right now, I hate it cause long press on back doesn't work, and it's what I use to kill apps

![Screenshot_20221118-131243_Bromite](https://user-images.githubusercontent.com/105613369/202784564-6a1b26cd-e68e-46a7-8784-9c5777bceaef.png)

**Comment by Amby713** [comment]
I formatted data and flashed crdroid 8.10 with MTGapps. This will always be the case when I say I flashed rom.

1. Flashed rom, signed into google account, it restored data (sms, call logs, contacts, system settings, etc) and it changed my DPI  (wm density 240) to what I had when I last backed up .
2.  Flashed rom, skipped logging into account ergo skipping data restore. Enabled adb debugging and used my laptop to access shell and set density to 240 (adb shell wm density 240) and it still had the weird positioning of the unlock pattern. Nav Bar was also jacked up, (see picture on last reply).

I'm not understanding how it was perfect on the first clean update from  8.09 to 8.10. It was perfect and had I not tried to troubleshoot the usb audio, it'd would have been perfect. I know there's a way this could be done, but can't figure it out.
